watch-site.js

```diff
@@ -0,0 +1,54 @@
+const chokidar = require('chokidar')
+const bs = require("browser-sync").create()
+const generator = require('mini-book-site-generator')
```

Comment:
I'm not sure that 'mini-book' is the appropriate prefix for this project. I'd like to think it can be for anyone writing Asciidoctor-based books. `asciidoctor-book-site-generator` might be better. Maybe "antora" would be OK, but I tend to think that asciidoctor is still the thing most people think of.

Reply:
> I'm not sure that 'mini-book' is the appropriate prefix for this project. I'd like to think it can be for anyone writing Asciidoctor-based books.

In a way that's true but it was still designed to produce mini-book site.
In practice, anyone writing AsciiDoc-based books could use it but they would probably have new requirements that are not supported/implemented.
So I don't want to promote it as a general purpose generator as it might be a little deceptive.

> asciidoctor-book-site-generator might be better. Maybe "antora" would be OK, but I tend to think that asciidoctor is still the thing most people think of.

Both Antora and Asciidoctor are implementation details. Even if we heavily rely on Antora, I don't think it adds value to add antora in the name.
We shouldn't use Asciidoctor because Antora will possibly use another AsciiDoc JavaScript implementation in the future... in fact, it's almost certain!

AsciiDoc is now an Eclipse Trademark so we cannot use asciidoc either.